### ISSUE TITLE
Multiple image upload takes long time

### ISSUE THREAD

**Issue by dragonfly91** [GospelAid/badiup#67]
Either add progress OR upload images in parallel. 

@Paglinawan please share your insights from a design POV.

**Comment by dragonfly91** [comment]
This is partly because images are being re-uploaded during edit. This has to be avoided. Filed a separate issue to track this. https://github.com/GospelAid/badiup/issues/93
